backend/trafficrules/src/main/java/com/coffeejug/trafficrules/controller/StatisticController.java

```diff
@@ -0,0 +1,21 @@
+package com.coffeejug.trafficrules.controller;
+
+import com.coffeejug.trafficrules.service.StatisticService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController
+@RequestMapping("/api/v1/statistic")

```

Comment:
Since we are using `users` (plural) as an URL for users let's use the plural here as well